oeps/architectural-decisions/oep-0052-arch-event-bus-architecture.rst

```diff
@@ -86,6 +90,8 @@ Decision
 .. _ADR on Kafka Managed Hosting: https://github.com/openedx/event-bus-kafka/blob/main/docs/decisions/0004-kafka-managed-hosting.rst
 .. _ADR on Managing Kafka Consumers: https://github.com/openedx/event-bus-kafka/blob/main/docs/decisions/0003-managing-kafka-consumers.rst
 
+.. _ADR on Redis streams Event Bus: https://github.com/openedx/event-bus-redis/blob/main/docs/decisions/0002-redis-streams-based-event-bus.rst
```

Comment:
Currently the ADR has not been merged, so its link is: https://github.com/openedx/event-bus-redis/pull/21/files